azure_jumpstart_arcbox/artifacts/installArcAgentUbuntu.sh

```diff
@@ -14,7 +14,7 @@ bash ~/install_linux_azcmagent.sh # 2>/dev/null
 ArcServerResourceName=$(hostname |sed -e "s/\b\(.\)/\u\1/g")
 
 # Run connect command
-azcmagent connect --access-token $accessToken --resource-group $resourceGroup --tenant-id $tenantId --location $Azurelocation --subscription-id $subscriptionId --resource-name "${ArcServerResourceName}" --cloud "AzureCloud" --tags "Project=jumpstart_arcbox" --correlation-id "d009f5dd-dba8-4ac7-bac9-b54ef3a6671a"
+azcmagent connect --access-token $accessToken --resource-group $resourceGroup --tenant-id $tenantId --location $Azurelocation --subscription-id $subscriptionId --resource-name "${ArcServerResourceName}" --cloud "AzureCloud" --correlation-id "d009f5dd-dba8-4ac7-bac9-b54ef3a6671a"
```

Comment:
Should the --tags parameter be specified here as well?

Reply:
The tags are applied using policy, so the machine would pick them up without setting them explicitly